core/src/main/scala/kafka/server/BrokerMetadataCheckpoint.scala

```diff
@@ -157,7 +157,8 @@ case class MetaProperties(
 object BrokerMetadataCheckpoint extends Logging {
   def getBrokerMetadataAndOfflineDirs(
     logDirs: collection.Seq[String],
-    ignoreMissing: Boolean
+    ignoreMissing: Boolean,
+    kraftMode: Boolean = false
```

Comment:
Default arguments like this make it easy to miss places that should be updated. Better to be explicit.

Reply:
Makes sense.